converters/fromZigbee.js

```diff
@@ -3128,6 +3128,13 @@ const converters = {
             };
         },
     },
+    dimmer_passthru_level: {
+        cid: 'genLevelCtrl',
+        type: 'cmdMoveToLevelWithOnOff',
+        convert: (model, msg, publish, options) => {
+            return {action: 'setlevel', level: msg.data.data.level};
```

Comment:
Could you change `level` to `brightness`? This make it's more consistent with other devices (e.g. the Hue remote dimmer).